appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -188,4 +190,30 @@ export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWiza
             }
         ];
     }
+
+    private getWorkflowStandardSkus(): WebSiteManagementModels.SkuDescription[] {
```

Comment:
Do we need to localize any display name?

Reply:
I dont see any value that is localizable here, following same pattern as Elastic Premium. I am assuming description is what we localize, which we dont have here.

Reply:
You can optionally add a label property which should be localized, otherwise we default to the name property which doesn't need to be localized.

Reply:
yeah not localizing the name here.